src/share/poudriere/jail.sh

```diff
@@ -924,6 +954,8 @@ create_jail() {
 	jset ${JAILNAME} method ${METHOD}
 	[ -n "${FCT}" ] && ${FCT} version_extra
 
+	jset ${JAILNAME} pkgbase ${BUILD_PKGBASE}
```

Comment:
Might as well extend `jail -i` to show this value in `info_jail`. Possibly `jail -l` too in `list_jail`

Reply:
I can do that in a later patch yes.

Reply:
Done